cmd/aries-agent-rest/startcmd/start.go

```diff
@@ -46,12 +52,25 @@ const (
 	agentTokenFlagUsage     = "Check for bearer token in the authorization header (optional)." +
 		" Alternatively, this can be set with the following environment variable: " + agentTokenEnvKey
 
-	// db path flag
-	agentDBPathFlagName      = "db-path"
-	agentDBPathEnvKey        = "ARIESD_DB_PATH"
-	agentDBPathFlagShorthand = "d"
-	agentDBPathFlagUsage     = "Path to database." +
-		" Alternatively, this can be set with the following environment variable: " + agentDBPathEnvKey
+	databaseTypeFlagName      = "database-type"
+	databaseTypeEnvKey        = "ARIESD_DATABASE_TYPE"
+	databaseTypeFlagShorthand = "q"
+	databaseTypeFlagUsage     = "The type of database to use for everything except key storage. " +
+		"Supported options: mem, couchdb, mysql. " +
+		" Alternatively, this can be set with the following environment variable: " + databaseTypeEnvKey
```

Comment:
We have `leveldb` as well in the sdk. Don't we need to support that as well ?

Reply:
I will add it in next PR

Reply:
I add it